test/unit/math/mix/scal/meta/broadcast_array_test.cpp

```diff
@@ -1,6 +1,7 @@
 #include <gtest/gtest.h>
 #include <stan/math/fwd/core/fvar.hpp>
 #include <stan/math/rev/core/var.hpp>
+#include <stan/math/rev/core/init_chainablestack.hpp>
```

Comment:
@wds15, do you know why this test needs this when the rest don't? It looks like the includes for the test don't actually have a mix header in there; maybe that would fix it?

Reply:
mix headers probably fix it, yes (but I am not familiar with the mix stuff). It looks to me as if the test does not include the right includes, yes. I do not know mix stuff at all which is why I plugged in the init_chainablestack.hpp.

Reply:
Ok. I made an educated guess what the includes should be... but have a look - I am not familiar with mix.